base/libc.jl

```diff
@@ -10,6 +10,9 @@ import Core.Intrinsics: bitcast
 
 export FILE, TmStruct, strftime, strptime, getpid, gethostname, free, malloc, calloc, realloc,
     errno, strerror, flush_cstdio, systemsleep, time, transcode
+if Sys.isunix()
```

Comment:
I think this can be unconditional

Reply:
I thought to make it consistent with

https://github.com/JuliaLang/julia/blob/4fa8055225dabd6619f6e70a10c1e36cb2282cc9/base/libc.jl#L16-L18

just below.  Also, it'd make `mkfi` <kbd>TAB</kbd> completes `mkfifo` in Windows while accessing to it throws:

```julia
julia> module Demo
       export foo
       end
Main.Demo

julia> using .Demo

julia> fo  # TAB
foldl   foldr    foo      for      foreach
julia> foo
ERROR: UndefVarError: foo not defined
```

So, I thought it's better to hide it in Windows.

Reply:
It's defined though, just has no methods.

Reply:
Ah, yes, you are right.  But what about the consistency with `GetLastError` and `FormatMessage`?  I'm OK with unconditional `export` but I just don't see a big reason to break the pattern.

Reply:
I'm okay with those being different